packages/quantic/force-app/main/default/lwc/quanticResultLink/quanticResultLink.js

```diff
@@ -90,10 +99,47 @@ export default class QuanticResultLink extends LightningElement {
     );
   };
 
+  checkResultType() {
+    if (this.result?.raw?.sfid !== undefined) {
+      this.isSalesforceLink = true;
+    }
+  }
+
+  handleOpenTabSalesforceLink(event) {
+    event.stopPropagation();
+
+    const targetPageRef = {
+      type: 'standard__recordPage',
+      attributes: {
+        recordId: this.getRecordIdAttribute(),
+        objectApiName: this.getObjectApiNameAttribute(),
```

Comment:
I've tested it and you should be able to remove this `objectApiName`
Test it with knowledge as well, but basically because we are providing Salesforce with a `recordId` the `objectApiName` seems to be irrelevant?

All in all, we can't rely on the `objecttype` from our fields, because people tend to change that, or to use the "label" of the object name instead of the API name.

Reply:
```suggestion
        
```

Reply:
yeah double checked with knowledge articles, seems to work fine